recipes/python-suitesparse-graphblas/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "python-suitesparse-graphblas" %}
+{% set version = "4.0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/GraphBLAS/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: db959dd4f6bef3839e2b2e600e7be287bf07aa289c893662291a184363b89878
+
+build:
+  skip: true  # [py<37]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - setuptools
+    - cffi
+    - cython
+    - graphblas
+    - numpy >=1.15
```

To use global pinning
```suggestion
    - numpy
```